dart/lib/src/sentry_span_interface.dart

```diff
@@ -55,6 +56,9 @@ abstract class ISentrySpan {
   @internal
   bool? get sampled;
 
+  @internal
+  void Function()? finishedCallback;
```

Comment:
we could technically get it via ctor and not expose here

Reply:
We also need to add it as optional param to the `startChild...` methods.

Reply:
We also have to expose it to all the `startChild` methods of spans.